specification/http/src/main/scripts/org/kaazing/specification/http/rfc7232/preconditions/if.match/strong.etag.get.status.412/request.rpt

```diff
@@ -5,18 +5,21 @@ write method "GET"
 write version "HTTP/1.1"
 write header content-length
 write flush
+write close
 
 read status "200" "OK"
 read header "ETag" /.+/
+read closed
 
 connect http://localhost:8000/resource
 connected
 
 write method "GET"
 write version "HTTP/1.1"
-write header "If-Match" "invalidetag"
+write header host
+write header "If-Match" "invalidEtag"
```

Comment:
Is this ETag invalid? Just wondering if it's syntactically invalid or just unexpected. If the RFC uses the terminology "invalid", then that's fine and probably better to be consistent anyway.